manifests/repo/mariadb.pp

```diff
@@ -0,0 +1,40 @@
+# Class: vision_mysql::repo::mariadb
+# ===========================
+#
+# Parameters
+# ----------
+#
+# Examples
+# --------
+#
+# @example
+# contain ::vision_mysql::repo::mariadb
+#
+
+class vision_mysql::repo::mariadb (
+
+) {
+
+  contain apt
+
+  $location = 'http://nyc2.mirrors.digitalocean.com/mariadb/repo/10.3/debian'
```

Comment:
Vielleicht besser den Server hier nehmen?
http://downloads.mariadb.com/MariaDB/mariadb-10.4/repo/debian/

Sieht etwas offizieller aus http://downloads.mariadb.com/MariaDB/

Außerdem: wieso ist in der $location MariaDB 10.3 hart codiert?

Reply:
Ich gebe zu ich hab das relativ salopp in der Uni geschrieben :sleeping:

Reply:
Kein Thema, dafür ist ja eine Review da :-)
Außerdem war das ja das EInzige was ich gefunden habe.
Der Rest sieht gut aus.

(Haben wir schon einen konkreten Anwendungsfall?)

Reply:
Neues Intranet klappt nicht mit 5.5